connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/util/IntegrationTestUtils.scala

```diff
@@ -57,6 +57,16 @@ object IntegrationTestUtils {
     Files.exists(Paths.get(filePath))
   }
 
+  private[sql] lazy val protobufJarPath: Option[File] = {
+    try {
+      Some(findJar("connector/protobuf", "spark-protobuf", "spark-protobuf"))
+    } catch {
+      case _: AssertionError => None
```

Comment:
What is throwing the assertion error?

Reply:
https://github.com/apache/spark/blob/c5db391b5d8447fe1384f7f957e4f200801c6823/connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/util/IntegrationTestUtils.scala#L93-L113

When `targetDir` not exists or `spark-protobuf` jar not exists

Reply:
For example, test with maven, but  the `connector/protobuf` module was not built first

Reply:
> For example, test with maven, but the `connector/protobuf` module was not built first

Hmm... Let me think again

Reply:
I removed this part due to `build/mvn clean install -pl connector/connect/server -am -DskipTests` will cascade build protobuf module